collect_app/src/main/res/values/strings.xml

```diff
@@ -314,32 +314,51 @@
     <!-- Geo string added in feature-geofeatures branch-->
 
     <string name="maps">Maps</string>
-    <string name="map_sdk_selector_title">Mapping SDK</string>
-    <string name="map_sdk_behavior">Mapping SDK</string>
-    <string name="google_maps_sdk">Google Maps SDK</string>
-    <string name="openstreetmap_sdk">OpenStreetMap SDK</string>
-    <string name="mapbox_maps_sdk">Mapbox Maps SDK</string>
-    <string name="show_map_sdk">Show Map SDK Options</string>
-    <string name="openmap_streets">OSM Streets</string>
     <string name="openmap_usgs_topo">USGS National Map Topo</string>
     <string name="openmap_usgs_sat">USGS National Map Hybrid</string>
     <string name="openmap_usgs_img">USGS National Map Imagery</string>
     <string name="openmap_stamen_terrain">Stamen Terrain</string>
     <string name="openmap_cartodb_positron">CartoDB Positron</string>
     <string name="openmap_cartodb_darkmatter">CartoDB Dark Matter</string>
-    <string name="streets">Streets</string>
-    <string name="terrain">Terrain</string>
-    <string name="hybrid">Hybrid</string>
-    <string name="satellite">Satellite</string>
-    <string name="map_basemap_selector_title">Basemaps</string>
-    <string name="show_map_basemap">Show Basemap Options</string>
 
-    <string name="mapbox_streets">Mapbox Streets</string>
-    <string name="mapbox_light">Mapbox Light</string>
-    <string name="mapbox_dark">Mapbox Dark</string>
-    <string name="mapbox_satellite">Mapbox Satellite</string>
-    <string name="mapbox_satellite_streets">Mapbox Satellite Streets</string>
-    <string name="mapbox_outdoors">Mapbox Outdoors</string>
+    <string name="base_layer">Base layer</string>
+    <string name="base_layer_source">Source</string>
+    <string name="base_layer_source_google">Google</string>
+    <string name="base_layer_source_mapbox">Mapbox</string>
+    <string name="base_layer_source_osm">OpenStreetMap</string>
+    <string name="base_layer_source_usgs">USGS</string>
+    <string name="base_layer_source_stamen">Stamen</string>
+    <string name="base_layer_source_carto">Carto</string>
+
+    <string name="google_map_style">Google map style</string>
+    <string name="mapbox_map_style">Mapbox map style</string>
+    <string name="usgs_map_style">USGS map style</string>
+    <string name="carto_map_style">Carto map style</string>
+
+    <string name="google_map_style_streets">Streets</string>
```

Comment:
Another one that was kicking around my brain -- since the style strings are being changed and will need to be translated anyway, I think we should take a moment to confirm they make as much sense as possible. I'm happy to do a follow-on issue/PR if it feels like a distraction.

Here's what I'm thinking:
- pull out "generic" style names that can be reused. That is, "Streets", "Terrain", "Satellite", "Hybrid" are the same regardless of the source. They should always be represented by the same string. They can be `map_style_streets`, etc.
- for the USGS maps, I would drop the "National" and expand "Topo" to "Topographic" as a generic style name. I'd also use the standard "Satellite" instead of "Imagery." So USGS would have three standard style options: Topographic, Satellite and Hybrid.
- Mapbox, Carto, Stamen would continue to have custom names.

Reply:
All makes sense!  I'm trying to prioritize getting you readable commits for now (as I'll be offline for the next couple of days), and then maybe I can do this after (on this PR or another)?

Reply:
That sounds perfect.

Reply:
This is now done.  I took the liberty of labelling "Mapbox Satellite Streets" as "Hybrid" as well, since that's what the "Hybrid" option does for all the other basemap sources.